package.json

```diff
@@ -75,6 +75,26 @@
         ],
         "configuration": "./language-configuration.json"
       },
+      {
+        "id": "terraform-stacks",
```

Comment:
Why is the language identifier for *.tfstack.hcl files terraform-stacks (with a plural s) and the one for *.tfdeploy.hcl just terraform-deployment (singular)?

I don't have any strong reason to ask other than that this might be last chance for us to fix an accidental inconsistency 😄

Reply:
Instead of attempting to reconcile the different pluralizations of the product name vs the language name for each of these files for what is essentially an internal facing identifier, I removed the plural and changed deployment to deploy.

The aliases are the public facing text, and can be changed as the product cements how it's referenced

Reply:
Sounds good! One thing I just asked myself: The files use `*.tfdeploy.hcl` and `*.tfstack.hcl` – why don't we use `tfdeploy` and `tfstack` as language identifiers 🤔 Would be very close to the suffix and a bit shorter than `terraform-deploy` or `terraform-stack`

Reply:
So far we've established `terraform` for `*.tf` files and `terraform-vars` for `*.tfvars` files. These language IDs are used in all editors that have LSP support. I like James' suggestion and think `terraform-deploy` and `terraform-stack` are reasonable choices to continue the established pattern.
It is also easier to decude the product name/context from the `terraform-*` rather than having to know what `tf` stands for.

Reply:
Makes sense! Thanks for the explanation!